packages/analytics/analytics-chart/README.md

```diff
@@ -510,3 +510,11 @@ const fileName = ref('exportFilename')
 </script>
 
 ```
+
+## Development
+
+Follow the setup instructions for the [top level README](../../../README.md). Then run the sandbox with:
+
+```
+pnpm run dev
```

Comment:
I think this will only work if you're in the `analytics-chart` package directory. 

If you're at the root the right command would be 
`pnpm --filter @kong-ui-public/analytics-chart run dev`

Reply:
The main readme mentions it
<img width="858" alt="image" src="https://github.com/user-attachments/assets/f42e3ad3-9f63-4774-8115-f348a2d85fed" />

Reply:
deleted my readme changes